### DISCUSSION TITLE
Few quick suggestions

### DISCUSSION THREAD

**Discussion by Turkey9002** [ACGaming/UniversalTweaks#348] [Tweak Suggestions]
- [x] Block Dispenser - Allows dispensers to place blocks.
https://www.curseforge.com/minecraft/mc-mods/blockdispenser

- [x] Easy Breeding - Dropped breeding items breeds animals.
https://www.curseforge.com/minecraft/mc-mods/easy-breeding

- [x] Entity Desync Fix - Fixes entity motion sync.
https://www.curseforge.com/minecraft/mc-mods/entity-desync-fix

- [x] Fast Workbench - Improves crafting table performance.
https://www.curseforge.com/minecraft/mc-mods/fastworkbench

- [x] Frame Void Patch - Fixes a bug with item frames.
https://www.curseforge.com/minecraft/mc-mods/frame-void-patch-mc-59363

- [ ] Friendermite - Endermen won't attack Endermites spawned from ender pearls anymore. 
https://www.curseforge.com/minecraft/mc-mods/friendermite

- [ ] Just Sleep - Makes setting spawn points on beds optional if there's already one setup.
https://www.curseforge.com/minecraft/mc-mods/just-sleep

- [ ] ~~Loading Progress Bar - Restores the world loading bar from older versions.
https://www.curseforge.com/minecraft/mc-mods/loading-progress-bar~~

- [ ] Mite2Shulker - Endermites can burrow into purpur blocks to form Shulkers. 
https://www.curseforge.com/minecraft/mc-mods/mite2shulker

- [ ] ~~Reach Fix - Fixes problems with entity hit detection.
https://www.curseforge.com/minecraft/mc-mods/reach-fix~~

- [ ] Sit - Allows you to sit on slabs and stairs by right-clicking on top of them.
https://www.curseforge.com/minecraft/mc-mods/sit

- [x] Villager Mantle Fix - Restores the unused villager hoods. (definitely won't be compatible with any mod that backports the 1.14 villagers though)
https://www.curseforge.com/minecraft/mc-mods/villager-mantle-fix

**Comment by ACGaming** [comment]
Valid suggestions!

**Comment by jbredwards** [comment]
I would advise against adding loading progress bar for now, since I'm still maintaining that mod and am planning on updating it when I'm less busy with my other projects :)

**Comment by ACGaming** [comment]
Reasonable, I‘m aiming to integrate more abandoned mods either way. 🙂

**Comment by Lylythii** [comment]
To build on this, if you're including FastWorkbench, you might want to consider FastFurnace;
https://www.curseforge.com/minecraft/mc-mods/fastfurnace

**Comment by Turkey9002** [comment]
I did actually thought of FastFurnace before but then realized LoliASM optimizes furnace recipes already. They're still compatible with each other though (with a slight benefit) so I think it be good to include too.

**Comment by ACGaming** [comment]
Just a quick heads-up, there‘s a master list for all requested mod features now where you can see which are already on my radar along with their current state: https://github.com/ACGaming/UniversalTweaks/wiki/Requested-Features

**Comment by ISurrealI** [comment]
> To build on this, if you're including FastWorkbench, you might want to consider FastFurnace; https://www.curseforge.com/minecraft/mc-mods/fastfurnace

CensoredASM includes it.

**Comment by Lylythii** [comment]
> > To build on this, if you're including FastWorkbench, you might want to consider FastFurnace; https://www.curseforge.com/minecraft/mc-mods/fastfurnace
> 
> CensoredASM includes it.

Sweet! Hadn't realised. That works for me.

**Comment by Lonelyxiya** [comment]
https://www.curseforge.com/minecraft/mc-mods/rlcombat 
rlcombat requires a reach fix prefix. But if UniversalTweaks integrates a reach fix, is it compatible with rlcombat? This is just an example.

**Comment by ACGaming** [comment]
Not too keen on integrating ReachFix right now since it‘s still maintained.

**Comment by Desenrrollado** [comment]
Outta curiosity is something like this on the radar or has it already been implemented? 

https://www.curseforge.com/minecraft/mc-mods/stimmedcow-nomorerecipeconflict/files

Some mods share different recipes for the same item and overlap. I know certain mods allow changing it. I think like craft tweaker. But it's still a neat feature especially since it hasn't been updated since 2019. With the redone version in the comments being kinda inactive and issues existing since 2021 still.

Not to mention adjustments and tweaks for it to change recipes seamlessly or save between crafts. There's a couple more for sure. But you've been implementing various and I appreciate it. Makes this mod a Swiss army knife and I appreciate the almost daily updates.

**Comment by ACGaming** [comment]
Thanks for the kind words!
Including a recipe conflict tweak is definitely planned and already on the [list of requested features](https://github.com/ACGaming/UniversalTweaks/wiki/Requested-Features). 🙂

**Comment by Desenrrollado** [comment]
What about implementing this as a tweak https://www.curseforge.com/minecraft/mc-mods/respawning-animals
I definitely don't like having to go out after awhile and just mob scarcity. Especially if you end up wiping out the local wild life population.
